doc/index.rst

```diff
@@ -36,11 +36,6 @@ Currently, PennyLane-Forest provides these Forest devices for PennyLane:
     :description: Forest's QVM and pyQuil pyQVM simulator.
     :link: devices/qvm.html
 
-.. devicegalleryitem::
```

Comment:
We also mention QPUs once in the README:
> Contains the PennyLane Forest plugin. This plugin allows different Rigetti devices to work with PennyLane — the wavefunction simulator, the Quantum Virtual Machine (QVM), and Quantum Processing Units (QPUs).

Could just remove the last few words?

Reply:
Good catch, thanks! Updated, also removed the device such that it's not provided by the plugin